README.md

```diff
@@ -53,38 +49,75 @@ Example
+
+    ```rust
+    // Example
+    env::set_var("BELLMAN_CUSTOM_GPU", "GeForce RTX 2080 Ti:4352, GeForce GTX 1060:1280");
+    ```
+
+- `BELLMAN_CPU_UTILIZATION`
 
-Currently only Nvidia hardware is supported, see [issue](https://github.com/finalitylabs/bellman/issues/3). Depending on the size of the proof being passed to the gpu for work, certain cards will not be able to allocate enough memory to either the FFT or Multiexp kernel. Below are a list of devices that work for small sets. In the future we will add the cuttoff point at which a given card will not be able to allocate enough memory to utilize the GPU.
+    Can be set in the interval [0,1] to designate a proportion of the multiexponenation calculation to be moved to cpu in parallel to the GPU to keep all hardware occupied.
 
+    ```rust
+    // Example
+    env::set_var("BELLMAN_CPU_UTILIZATION", "0.5");
+    ```
+
+#### Supported / Tested Cards
+
+Depending on the size of the proof being passed to the gpu for work, certain cards will not be able to allocate enough memory to either the FFT or Multiexp kernel. Below are a list of devices that work for small sets. In the future we will add the cuttoff point at which a given card will not be able to allocate enough memory to utilize the GPU.
+
+| Device Name            | Cores | Comments       |
+|------------------------|-------|----------------|
+| TITAN RTX              | 4608  |                |
+| Tesla V100             | 5120  |                |
+| Tesla P100             | 3584  |                |
+| Tesla T4               | 2560  |                |
+| GeForce RTX 2080 Ti    | 4352  |                |
+| GeForce RTX 2080 SUPER | 3072  |                |
+| GeForce RTX 2080       | 2944  |                |
+| GeForce RTX 2070 SUPER | 2560  |                |
+| GeForce GTX 1080 Ti    | 3584  |                |
+| GeForce GTX 1080       | 2560  |                |
+| GeForce GTX 2060       | 1920  |                |
+| GeForce GTX 1660 Ti    | 1536  |                |
+| GeForce GTX 1060       | 1280  |                |
+| GeForce GTX 1650 SUPER | 1280  |                |
+| GeForce GTX 1650       | 896   |                |
+|                        |       |                |
+| gfx1010                | 2560  | AMD RX 5700 XT |
+
+### Running Tests
+
+To run the multiexp_consistency test you can use:
+
+```bash
+RUST_LOG=info cargo test --features gpu -- --exact multiexp::gpu_multiexp_consistency --nocapture
 ```
-("Device_Name", Cores),
-("Quadro RTX 6000", 4608),
-("TITAN RTX", 4608),
-("Tesla V100", 5120),
-("Tesla P100", 3584),
-("Tesla T4", 2560),
-("Quadro M5000", 2048),
-("GeForce RTX 2080 Ti", 4352),
-("GeForce RTX 2080 SUPER", 3072),
-("GeForce RTX 2080", 2944),
-("GeForce RTX 2070 SUPER", 2560),
-("GeForce GTX 1080 Ti", 3584),
-("GeForce GTX 1080", 2560),
-("GeForce GTX 2060", 1920),
-("GeForce GTX 1660 Ti", 1536),
-("GeForce GTX 1060", 1280),
-("GeForce GTX 1650 SUPER", 1280),
-("GeForce GTX 1650", 896),
```

Comment:
I assume you did not intentionally remove cards from this list. Can you update to re-include the ones which were removed?

Reply:
done

Reply:
seems wrong to have this duplicated, if this information now lives in the table above

Reply:
yes, sure. I added back the Quadro boards in the table above